nucypher/config/base.py

```diff
@@ -420,7 +420,7 @@ def __init__(
         self.crypto_power = crypto_power
         if keystore_path and not keystore:
             keystore = Keystore(keystore_path=keystore_path)
-        self.__keystore = self.__keystore = keystore or NO_KEYSTORE_ATTACHED.bool_value(
+        self.__keystore = keystore or NO_KEYSTORE_ATTACHED.bool_value(
             False
         )
```

Comment:
To appease the linter - https://github.com/nucypher/nucypher/actions/runs/13754840273/job/38632877610?pr=3580#step:4:156

```suggestion
        self.__keystore = keystore or NO_KEYSTORE_ATTACHED.bool_value(False)
```